File: Classes/Banco/src/Conta.java
public class Conta {
    String titular;
    String numero;
    double saldo;

    void depositar(double valor) {
        this.saldo += valor;
        System.out.println("Depositado o valor de " + valor + " na conta de " + this.titular);
        System.out.println("Novo saldo " + this.saldo);
    }

    void sacar(double valor) {
        if (valor <= this.saldo) {
            this.saldo -= valor;
            System.out.println("Retirado o valor de " + valor + " na conta de " + this.titular);
            System.out.println("Novo saldo " + this.saldo);
        } else {
            System.out.println("Não é possível realizar o saque, pois o valor solicitado é maior que saldo disponível");
            System.out.println("Saldo disponível: " + this.saldo);
        }
    }
    void transferir(double valor, Conta contaDestino){
        if (valor <= this.saldo) {
            this.saldo -= valor;
            System.out.println("Retirado o valor de " + valor + " na conta de " + this.titular);
            System.out.println("Novo saldo " + this.saldo);
            contaDestino.depositar(valor);
        } else {
            System.out.println("Não é possível realizar o saque, pois o valor solicitado é maior que saldo disponível");
            System.out.println("Saldo disponível: " + this.saldo);
        }
    }

}

File: Classes/Banco em Python/conta.py
class Conta:
    titular = ""
    numero = ""
    saldo = 0.0

    def depositar(self, valor):
        self.saldo += valor
        print(f"Valor depositado de {valor} na conta de {self.titular}")
        print(f"Novo saldo {self.saldo}")

    def sacar(self, valor):
        if valor <= self.saldo:
            self.saldo -= valor
            print(f"Valor retirado de {valor} na conta de {self.titular}")
            print(f"Novo saldo {self.saldo}")

        else:
            print("Não foi possível efetuar o saque.")
            print(f"Saldo disponível: {self.saldo}")

    def transferir(self, valor, conta_destino):
        if valor <= self.saldo:
            self.saldo -= valor
            print(f"Valor retirado de {valor} na conta de {self.titular}")
            print(f"Novo saldo {self.saldo}")
            conta_destino.depositar(valor)

        else:
            print("Não foi possível efetuar o saque.")
            print(f"Saldo disponível: {self.saldo}")
